packages/plugin-ext/package.json

```diff
@@ -78,6 +79,7 @@
     "@theia/ext-scripts": "^0.14.0",
     "@types/decompress": "^4.2.2",
     "@types/escape-html": "^0.0.20",
+    "@types/fs-extra": "^4.0.2",
```

Comment:
Why do you add "@types/fs-extra": "^4.0.2", in devDependencies when you already included it in dependencies ?
Not needed here.

Reply:
I don't see the duplication. Typings are added as devdeps, with the actual thing as a dep.

Reply:
Hello.

The fs-extra package does not include type signatures. Therefore the respective @types package is consumed as a dev dep.

Fs extra is already at version 8. However i did not want to introduce a new dep as part if this change so i used a version which is already consumed else where in theia.

Reply:
Hello.

The fs-extra package does not include type signatures. Therefore the respective @types package is consumed as a dev dep.

Fs extra is already at version 8. However i did not want to introduce a new dep as part if this change so i used a version which is already consumed else where in theia.